specs/scrabble.spec.js

```diff
@@ -181,44 +194,79 @@ describe('Player', () => {
   });
 
   describe('hasWon', () => {
-    test.skip('returns false when score < 100', () => {
-
+    test('returns false when score < 100', () => {
+      const player = new Scrabble.Player('test player');
+      const words = [{word: 'dog', score: 5}, {word: 'cat', score: 5}, {word: 'goat', score: 5}];
+      let totalScore = 0;
```

Comment:
I like how explicit you are here about the expected scores of these words. That makes this test very readable!